cli.py

```
@@ -0,0 +1,904 @@
+        if args.format:
+            file_format = args.format
+        else:
+            if input_file.suffix.lower() in ['.yaml', '.yml']:
+                file_format = 'yaml'
+            elif input_file.suffix.lower() == '.json':
+                file_format = 'json'
+            else:
+                raise CLIError("Could not determine file format. Use --format to specify.")
+        
+        # Load file
+        with open(input_file, 'r', encoding='utf-8') as f:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)